script/ynETH/VerifyYnETH.s.sol

```diff
@@ -0,0 +1,717 @@
+/// SPDX-License-Identifier: BSD 3-Clause License
+pragma solidity ^0.8.24;
```

Comment:
the verify ynETH is now ehances to check:


Proxy - ProxyAdmin - Implementation Trinity matches the JSON with what's on-chain.

All the deps and fields of withdrawals are asserted